script.js

```diff
@@ -0,0 +1,95 @@
+const initApp = () => {
+  const hamburger = document.querySelector('.hamburger');
+  const topLinks = document.querySelector('.top-links');
+
+  hamburger.addEventListener('click', () => {
+    hamburger.classList.toggle('active');
+    topLinks.classList.toggle('active');
+  });
+
+  document.querySelectorAll('.top-link').forEach((n) => n.addEventListener('click', () => {
+    hamburger.classList.remove('active');
+    topLinks.classList.remove('active');
+  }));
+
+  const speakers = [
+    {
+      name: 'Mohamed Gamaleldin',
+      title: 'Owner and Manager at The English Club',
+      image: './images/gamal.webp',
+      description:
+          'Createor of PCC programm, Presentation Competetion Winner in The British Council in Cairo 2017',
+    },
+    {
+      name: 'Mohamed Halim',
+      title: 'Owner and Manager at The English Club',
+      image: './images/halim.webp',
+      description: 'Creator of "PCC Plus". Presentation competition winner at the MODLI Institution in Alexandria 2018',
+    },
+    {
+      name: 'Asmaa El-sayed',
+      title: 'Manager at The English Club',
+      image: './images/asmaa.webp',
+      description:
+            'Facilitator of the PCC competition for over five consecutive years. Debate competition winner 2021',
+    },
+    {
+      name: 'PROF. Ali Atalla',
+      title: 'PCC main Judge and Language Teacher',
+      image: './images/ali.webp',
+      description:
+            'The Main Judge in the PCC for five consecutive years. The creator of PCC judging Criterea',
+    },
+    {
+      name: 'Adel Samy',
+      title: 'Arabic teacher for non-Arabic speakers.',
+      image: './images/adel.jpg',
+      description:
+            "Founder and judge at the first PCC. Plays an important role in Assessing student's roles",
+    },
+  ];
+
+  const speakersWrap = document.querySelector('.speakers .list');
+  for (let i = 0; i < speakers.length; i += 1) {
+    const speaker = speakers[i];
+    speakersWrap.innerHTML += `
+      <div class="speaker-card">
+        <div class="image">
+          <img src="${speaker.image}" alt="picture of ${speaker.name}" />
+        </div>
+        <div class="details">
+          <h4>${speaker.name}</h4>
+          <p class="description">${speaker.title}</p>
+          <hr />
+          <p>
+            ${speaker.description}
+          </p>
+        </div>
+      </div>
```

Comment:
- [ OPTIONAL] Great job fixing the required changes but I think you should change the color of the description and heading into black as shown in the given design so please change it 🤩🙏🏻⭐

- Your
![image](https://github.com/mohamedgamaleldin999999/mohamedgamaleldin999999.github.io/assets/58553711/a0c6f903-183f-4a40-9214-50dbc75656de)

- Given
![image](https://github.com/mohamedgamaleldin999999/mohamedgamaleldin999999.github.io/assets/58553711/220a8c57-7d47-4796-940f-e8a6932abb58)

Reply:
Thank you for assigning them as optional. I will apply them. I appreciate it.